examples/210318_gospecreading_assignability/rule4.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import "fmt"
+
+type T chan int // Tはchannel type
```

Comment:
この例だと「ルール2」でも代入可能であることが説明できてしまうので、ルール4の必要性の説明になっていないと思います。

このルール4はunderlying typeが一方向チャネルになっているときに初めて必要となるので、たとえば次のように変更するのはどうでしょうか？

```suggestion
type T <-chan int // Tはchannel type
```

Reply:
> この例だと「ルール2」でも代入可能であることが説明できてしまう

確かにですね🙏
提案していただいたとおり修正します
ありがとうございます〜